### PR TITLE
Update gtk crate version to 0.16.1

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: gtk/Cargo.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ name = "gtk"
 homepage = "https://gtk-rs.org/"
 authors = ["The gtk-rs Project Developers"]
 readme = "README.md"
-version = "0.16.0"
+version = "0.16.1"
```
Comment (sdroege) [inline gtk/Cargo.toml:10]: There are more crates in here. Best to update them all in sync (and only upload the ones that have actual changes). That's what we do elsewhere.

Comment (GuillaumeGomez) [reply gtk/Cargo.toml:10]: Done!

Comment (GuillaumeGomez) [reply gtk/Cargo.toml:10]: Not for the `sys` crates though, should I do it for them too?

Comment (sdroege) [reply gtk/Cargo.toml:10]: All

Comment (GuillaumeGomez) [reply gtk/Cargo.toml:10]: Done as well!

**Comment by sdroege** [comment]
CI needs fixing but otherwise seems OK

**Comment by GuillaumeGomez** [comment]
Should I fix it before merging this PR?

**Comment by sdroege** [comment]
That would be good :)

**Pull request merged**